CorsixTH/Lua/objects/couch.lua

```diff
@@ -54,11 +54,11 @@ object.usage_animations = copy_north_to_south {
 }
 object.orientations = {
   north = {
-    footprint = { {-1, -1, complete_cell = true}, {-1, 0, complete_cell = true}, {0, -1}, {0, 0, only_passable = true} },
+    footprint = { {-1, -1, complete_cell = true}, {-1, 0, complete_cell = true}, {0, -1, only_passable = true}, {0, 0, only_passable = true} },
```

Comment:
Just a sidenote: an object may already exist on this now "only_passable" cell. This pr doesn't account for such a case (because nothing breaks with the object there, currently)